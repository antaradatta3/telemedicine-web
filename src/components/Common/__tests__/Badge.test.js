import React from 'react';
import { mount } from 'enzyme';

import Badge from '../Badge';

describe('Component: Badge', () => {
  const props = {
    label: 'Sample',
    type: 'primary',
    count: 200,
  };
  it('Badge matches snapshot', () => {
    const component = mount(<Badge {...props} />);
    expect(component).toMatchSnapshot();
  });
});
