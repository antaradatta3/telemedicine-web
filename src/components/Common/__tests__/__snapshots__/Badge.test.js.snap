// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Badge Badge matches snapshot 1`] = `
<Badge
  count={200}
  label="Sample"
  type="primary"
>
  <span>
     
    <span
      className="badge badge-pill badge-primary"
    >
      200
    </span>
     
    Sample
      
  </span>
</Badge>
`;
